flush-print/progress.py

```diff
@@ -0,0 +1,21 @@
+from time import sleep
+
+
+def progress(percent=0, width=30):
+    left = width * percent // 100
+    right = width - left
+    print(
+        "\r[",
+        "#" * left,
+        " " * right,
+        "]",
+        f" {percent:.0f}%",
+        sep="",
+        end="",
+        flush=True,
+    )
```

Comment:
Nice example!

Feel free to disregard the following suggestion. I find something like this a bit more readable:

```suggestion
def progress(percent=0, width=30):
    progress = (width * percent // 100)
    bar = "#" * progress
    pad = " " * (width - progress)
    print(f"\r[{bar}{pad}]{percent:.0f}%", end="", flush=True)
```

Reply:
pretty rewrite! i had just taken this from bartosz print mega tutorial, but i like your code more. updated ✅

Reply:
Just added a space to make it look exactly the same as the other version